docs/sources/flow/reference/components/loki.write.md

```diff
@@ -154,3 +154,7 @@ loki.write "local" {
     }
 }
 ```
+
+## Technical details
```

Comment:
```suggestion
## File compression
```

@mattdurham Could we be more specific for the heading (and in all other topics here)?  Or are we planning to add other information beyond just file compression support?

Reply:
I was thinking it could be more generic and grow over time but that might be to forward looking. I will change `Compression` and we can expand later if needed.

Reply:
FYI I dont want to use file compression since its really line level compression

Reply:
The title makes perfect sense as just `Compression`.